README.md

```diff
@@ -84,38 +128,37 @@ On completion, you will see the following message:
 
 ![project appropriation success](readme-assets/project-appropriation-success.png)
 
-### Install the dependencies
-
-You can now install the dependencies:
-
-`npm i`
-
-### Run the whole stack
+### Run the application
 
-Running the database:
-
-`npm run postgres`
-
-Running the api:
-
-`npm start api`
-
-Running the webapp:
-
-`npm start webapp`
+First, install the dependencies:
+```
+npm i
+```
 
-Running all the tests:
+Then, run the whole stack:
+```
+npm run postgres
+```
+```
+npm start api
+```
+```
+npm start webapp
+```
 
-`npm run affected:test && npm run affected:e2e`
+Finally, why not test it:
+```
+npm run affected:test && npm run affected:e2e
+```
 
 For a full list of available commands, consult the `package.json`.
 
-### Github Actions
+### Continuous Integration
 
-This templates integrates Github Actions as Continuous Integration. The existing workflows are under `.github/workflows`. 
+This templates integrates Github Actions for its Continuous Integration. The existing workflows are under `.github/workflows`. 
 To have the CI working, you must:
-- Link your repository with [Coveralls](https://coveralls.io/repos/new).
-- (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
+1. Link your repository with [Coveralls](https://coveralls.io/repos/new).
+1. (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
```

Comment:
```suggestion
2. (Optional) Insert your [Nx Cloud](https://nx.app/) access token in github secrets under `NX_CLOUD_TOKEN`. This enables for caching and faster build times.
```

Reply:
Did you know, if you put only 1s, markdown generates the numbers automatically

Reply:
![](https://media.giphy.com/media/gVE7nURcnD9bW/giphy.gif)